Ownership/TC - LTC/ABBOT.txt

```diff
@@ -1,5 +1,5 @@
 SECTOR:London ABBOT stack:0:12500
-OWNER:TCE:LE:LC:LNC:LWC:LSC:L
+OWNER:TCDAG:TCRED:TCE:LE:LC:LNC:LWC:LSC:L
```

Comment:
I think REDFA should own the ABBOT stack, and not hand aircraft to Dagga. It appears to make more sense when London ABBOT Stack sector is turned on as well as Redfa's airspace.

![image](https://user-images.githubusercontent.com/12017284/64631806-c4911e00-d3ef-11e9-8f2d-adc5f63071fb.png)

Dagga only goes down to 155 in this area, but is then asked to handle London ABBOT stack up to 125 otherwise?